tutorial/notebook_setup.md

```diff
@@ -0,0 +1,71 @@
+# Launching the Jupyter notebooks
+
+The materials on this website are actually the tutorial notebooks. We encourage you
+to follow along with the workshop in a fresh, blank notebook. However, if you
+would like to be able to run the completed notebooks locally, you can use the instructions below.
+
+If you cloned the workshop repository, then you already have the notebooks in the `tutorial` subfolder.
+If you have not then you can download the notebooks as follows:
+
+### Cloning via git
```

Comment:
Is this intended to come before the `install_dependencies.md` file? Then we might have a problem, because our [tutorial page](https://cfp.scipy.org/2024/talk/PQMQ3K/) points at the `install_dependencies.md` as the first entry point. 

What you you think about inserting the content of this file as the first section in `install_dependencies.md`?

Reply:
For people to install the dependencies following the instructions they need the env.yaml so I copied these instructions to `install_dependencies.md`.
I dont think everyone will need the notebook setup instructions, so duplication seems fine?

Reply:
I'm somewhat against the duplication, because it makes for another paragraph newbies have to read through to decide if they need to follow it. 

Honestly, is there any reason not to include the rest of this content on launching the notebooks at the end of the `install_dependencies.md` too? Having this in another file doesn't really add much.

And launching the notebooks is something that we will demonstrate during the tutorial anyway.

Reply:
Sure!